### PR TITLE
Moving copy/pasted docs to shared istio/devel

### PR DESCRIPTION
**Pull request by ldemailly**
See https://github.com/istio/istio/pull/352

### CODE REVIEW

**Comment by ldemailly** [comment]
will let you force merge to skip over the prexisting
```
ERROR: No test targets were found, yet testing was requested.
```
jenkins failure

**Comment by xiaolanz** [comment]
please ignore the presubmit failure. it will not block your submission.

File: doc/dev/development.md [hunk 1]
```diff
@@ -1,327 +1,7 @@
 # Development Guide
```
Comment (geeknoid) [inline doc/dev/development.md]: I'd prefer that we delete this document and update the root-level README.md to point developers to istio/istio directly.

**Comment by xiaolanz** [comment]
+1, be careful there are more than one references to this doc/dev/
development.md.

Xiaolan (Joy) Zhang | Software Engineer | Google Inc. | xiaolan@google.com |
This email may be confidential or privileged.  If you received this
communication by mistake, please don't forward it to anyone else, please
erase all copies and attachments, and please let me know that it went to
the wrong person.

**Comment by ldemailly** [comment]
for this one `git grep`'ed for `development.md` and changed all occurences

PTAL

**Pull request merged**